galaxy_ng/app/tasks/promotion.py

```diff
@@ -25,3 +26,29 @@ def remove_content_from_repository(collection_version_pk, repository_pk):
     qs = CollectionVersion.objects.filter(pk=collection_version_pk)
     with repository.new_version() as new_version:
         new_version.remove_content(qs)
+
+
+def dispatch_copy_to_org_repos(collection_version_pk):
+    """Starts tasks to copy CollectionVersion to org repos.
+    Method to start tasks, not a task itself."""
+
+    # get all org distros
+    # TODO: replace with correct convention, regex in constants?
+    org_distros = AnsibleDistribution.objects.filter(name__startswith='org-')
```

Comment:
I guess looking for `org-` in the name is OK, since this is confined to c.rh.c

Reply:
In theory, looking up 'base_path'  would be more specific, but:

1) For now, Distro name and base_path will be the same.
2) I'm not 100% sure 'base_path' get's a db index though probably does, but name definitely should.